pom.xml

```diff
@@ -31,7 +31,7 @@
         <javax.transaction.version>1.3</javax.transaction.version>
         <xaPoolVersion>1.5.0</xaPoolVersion>
         <derbyVersion>10.11.1.1</derbyVersion>
-        <mySqlVersion>5.1.48</mySqlVersion>
+        <mySqlVersion>8.0.19</mySqlVersion>
```

Comment:
This will break the MySql logger: https://www.mulesoft.org/jira/browse/MULE-13944

Reply:
I have made the required changes to get it to work with the new version. Do you want me to push them and check them with someone else?